x11/xsetmode/files/apple-silicon.diff

```diff
@@ -0,0 +1,19 @@
+--- config.guess.orig	2020-09-07 13:47:04.000000000 -0400
++++ config.guess	2020-09-07 13:47:21.000000000 -0400
+@@ -3,7 +3,7 @@
+ #   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+ #   2000, 2001, 2002, 2003 Free Software Foundation, Inc.
+ 
+-timestamp='2003-06-17'
++timestamp='2020-09-07'
```

Comment:
same comments as the other PR:

(1) this change isn't technically necessary

(2) please consider an upstream PR for this fix